src/models/sunburst.js

```diff
@@ -144,48 +239,51 @@ nv.models.sunburst = function() {
                 d.dx0 = d.dx;
             }
 
+            //FIXME THIS WAS NEVER CALLED - updates arn't working though ... I couldn't figure out
+            //to do it
+            //
             // When switching data: interpolate the arcs in data space.
-            function arcTweenData(a, i) {
-                var oi = d3.interpolate({x: a.x0, dx: a.dx0}, a);
-
-                function tween(t) {
-                    var b = oi(t);
-                    a.x0 = b.x;
-                    a.dx0 = b.dx;
-                    return arc(b);
-                }
-
-                if (i == 0) {
-                    // If we are on the first arc, adjust the x domain to match the root node
-                    // at the current zoom level. (We only need to do this once.)
-                    var xd = d3.interpolate(x.domain(), [node.x, node.x + node.dx]);
-                    return function (t) {
-                        x.domain(xd(t));
-                        return tween(t);
-                    };
-                } else {
-                    return tween;
-                }
-            }
+            //   function arcTweenData(a, i) {
+            //     var oi = d3.interpolate({x: a.x0, dx: a.dx0}, a);
+            //
+            //     function tween(t) {
+            //       var b = oi(t);
+            //       a.x0 = b.x;
+            //       a.dx0 = b.dx;
+            //       return arc(b);
+            //     }
+            //
+            //     if (i == 0) {
+            //       // If we are on the first arc, adjust the x domain to match the root node
+            //       // at the current zoom level. (We only need to do this once.)
+            //       var xd = d3.interpolate(x.domain(), [node.x, node.x + node.dx]);
+            //       return function (t) {
+            //         x.domain(xd(t));
+            //         return tween(t);
+            //       };
+            //     } else {
+            //       return tween;
+            //     }
+            //   }
```

Comment:
I commented out the function because it wasn't called, but probably should in case the updating is triggered.